Entities/ACraft.cpp

```diff
@@ -806,6 +809,31 @@ bool ACraft::OnMOHit(MovableObject *pOtherMO)
 }
 
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+void ACraft::GibThis(Vector impactImpulse, float internalBlast, MovableObject* pIgnoreMO) {
+
```

Comment:
This is nitpicky, but please kill off the empty line at the top and bottom of this method.